src/components/MAPI/utils.ts

```diff
@@ -909,6 +909,12 @@ export function getClusterDescription(
   }
 }
 
+export function getClusterServicePriority(cluster: Cluster): string {
+  const labels = capiv1beta1.getClusterLabels(cluster);
+
+  return labels[capiv1beta1.labelServicePriority];
+}
+
```

Comment:
We have some functions for getting cluster labels in `src/model/services/mapi/capiv1beta1/key.ts` - do you think it makes sense to place this with those?